cognite_toolkit/_builtin_modules/contextualization/p_and_id_parser/functions/contextualization_connection_writer/handler.py

```diff
@@ -0,0 +1,249 @@
+import json
+from collections.abc import Iterable, Sequence
+from collections import defaultdict
+from typing import Literal, ClassVar, TypeVar
+
+import yaml
+from cognite.client import CogniteClient
+from cognite.client import data_modeling as dm
+from cognite.client.data_classes import ExtractionPipelineRunWrite, RowWrite
+from cognite.client.data_classes.data_modeling.cdm.v1 import CogniteDiagramAnnotationApply, CogniteDiagramAnnotation
+
+from pydantic import BaseModel
+from pydantic.alias_generators import to_camel
+
+FUNCTION_ID = "connection_writer"
+EXTRACTION_PIPELINE_EXTERNAL_ID = "p_and_id_parser"
+RAW_DATABASE = "contextualizationState"
```

Comment:
If it is hardcoded here, shouldn't it be hardcoded in the config? Or a variable both places? Having it as a variable elsewhere suggests that the user can change it, but the function would fail